src/styles/global.css

```
@@ -80,4 +80,124 @@ input[type="date"]::-webkit-calendar-picker-indicator {
+    border: 1px solid var(--border);
+    border-radius: 6px;
+  }
+
+  .navbar-right button:last-child:hover {
+    background: var(--surface) !important;
+  }
+
+  .navbar-right button:last-child svg {
+    margin: 0;
+    font-size: 16px;
+    fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)